lms/djangoapps/discussion/static/discussion/templates/fake-breadcrumbs.underscore

```diff
@@ -1,6 +1,8 @@
 <h6 class="hd-6 breadcrumbs">
     <span class="nav-item">
-        <a class="all-topics" href="">All Topics</a>
+        <a class="all-topics" href="">
+            <span class="fa fa-bars"></span>All Topics
```

Comment:
Is it the case that we need some a11y classes on the span containing the icon? aria-hidden perhaps?